Content.Shared/GameObjects/Components/Chemistry/SolutionComponent.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using Content.Shared.Chemistry;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Maths;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Shared.GameObjects.Components.Chemistry
+{
+    public class SolutionComponent : Component
```

Comment:
For clarity it might be better to name it something like `SolutionContainer` isntead.

Reply:
I avoided calling it a container because people might get the idea that it is only used for container-like entities (buckets, beakers). It will also be used for things like the human body blood, probably the oil/blood pools on the floor around the map, poisoned weapons, whatever liquids furry players need, etc.

The class name also follows the current naming convention of components, so idk if making this one a special snowflake would help.